blurring_as_a_service/performace_evaluation_pipeline/source/get_data.py

```diff
@@ -0,0 +1,8 @@
+from os import listdir
+
+
+def get_and_store_info(data_folder_path: str, output_file_path: str):
```

Comment:
Mmm a file for this seems an overkill. It's also not super clear on first read what it does.
Can we think if it belongs somewhere else or together with other stuff and make the purpose clearer?